CMakeLists.txt

```diff
@@ -52,11 +51,20 @@ if ("$ENV{TARGET_ARCH}" STREQUAL "arm")
     fatal_linker_warnings=false
   )
 else()
-  list(APPEND GN_GEN_ARGS
-    rtc_build_tools=false
-  )
+  if ("$ENV{TARGET_ARCH}" STREQUAL "arm64")
```

Comment:
This can be

```cmake
elseif("$ENV{TARGET_ARCH}" STREQUAL "arm64")
```

and save us a level of indention.

Reply:
Maybe a better option would be to do something like

```cmake
if("$ENV{TARGET_ARCH}" STREQUAL "arm" OR "$ENV{TARGET_ARCH}" STREQUAL "arm64")
  list(APPEND GN_GEN_ARGS
    target_os="linux"
    target_cpu="$ENV{TARGET_ARCH}"
    rtc_build_tools=true
    treat_warnings_as_errors=false
    fatal_linker_warnings=false
  )
else()
  # amd64 options
endif()
```

and collapse both arm and arm64 branches into one.